ci_handbook/continuous_integration.qmd

```diff
@@ -0,0 +1,12 @@
+---
+title: "{{< fa arrow-right >}} Continuous Integration & Delivery"
```

Comment:
Is this another way of saying 'Ecological Forecast Operationalization'
I don't know what is more commonly understood, but maybe 'Continuous Data Integration & Forecast Delivery' would be more explicit

Reply:
This is referring to software development practice of continuous integration and continuous delivery, https://en.wikipedia.org/wiki/CI/CD ; there are various tools and best practices for how to go about this and I'm imagining having an article dedicated to this for forecast CI.  I'll leave as-is for now